core/src/main/scala/org/http4s/sbt/Http4sOrgPlugin.scala

```diff
@@ -45,7 +45,7 @@ object Http4sOrgPlugin extends AutoPlugin {
 
   lazy val publishSettings: Seq[Setting[_]] =
     Seq(
-      tlSonatypeUseLegacyHost := false
+      ThisBuild / sonatypeCredentialHost := xerial.sbt.Sonatype.sonatypeLegacy
```

Comment:
We don't publish to the legacy host. Actually we can just delete this setting, it's no longer necessary.